apps-enable.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# This script assumes to be located in /IONOS as submodule within the Nextcloud server
+# repository.
+
+BDIR="$( dirname "${0}" )"
+
+NEXTCLOUD_DIR="${BDIR}/.."
+APPS_DIR="${NEXTCLOUD_DIR}/apps-external"
+
+ooc() {
+	php occ \
```

Comment:
If you'd change this to `php ${NEXTCLOUD_DIR}/occ` one could run it from / or /IONOS